tardis/tardis_portal/models/parameters.py

```diff
@@ -435,15 +438,53 @@ def set_value(self, value):
             self.numerical_value = float(value)
         elif self.name.isDateTime():
             # We convert the value string into datetime object.
-            # dateutil.parser detects and converts many date formats and is quite
-            # permissive in what it accepts (form validation and API input
-            # validation happens elsewhere and may be less permissive)
+            # dateutil.parser detects and converts many date formats and is
+            # quite permissive in what it accepts (form validation and API
+            # input validation happens elsewhere and may be less permissive)
             datevalue = dateutil.parser.parse(value)
             if settings.USE_TZ and is_naive(datevalue):
                 datevalue = make_aware(datevalue, LOCAL_TZ)
             elif not settings.USE_TZ and is_aware(datevalue):
                 datevalue = make_naive(datevalue, LOCAL_TZ)
             self.datetime_value = datevalue
+        elif self.name.isLink():
+            # Always store the raw value as a string, even if setting
+            # the GenericForeignKey via link_id/link_ct
+            self.string_value = unicode(value)
+
+            try:
+                # We detect experiment or dataset view URLs
+                # (eg, /experiment/view/12345 or /dataset/123)
```

Comment:
Add API example and required trailing slashes to comment: eg /exeriment/view/123/ or /api/v1/dataset/456